server/src/main/java/org/candlepin/pinsetter/tasks/HealEntireOrgJob.java

```diff
@@ -122,7 +122,7 @@ public static JobDetail healEntireOrg(Owner owner, Date entitleDate) {
         map.put(JobStatus.TARGET_TYPE, JobStatus.TargetType.OWNER);
```

Comment:
This file shouldn't be here at all, since the whole pinsetter package should be gone.